demo/src/pages/components.rs

```diff
@@ -76,7 +76,7 @@ impl IntoView for MenuGroupOption {
     fn into_view(self) -> View {
         let Self { label, children } = self;
         view! {
-            <Caption1Strong style="margin-inline-start: 10px">
+            <Caption1Strong style="margin-inline-start: 10px; margin-top: 10px; display: inline-block">
```

Comment:
Added this separation here because the layout looked confusing to me. The separation between sections was missing.